index.html

```diff
@@ -1028,6 +1061,454 @@ <h2>
+            configuration with an eccentric rotating mass (ERM) vibration
+            motor in each handle of a standard gamepad. In this
+            configuration, either motor is capable of vibrating the whole
+            gamepad. The vibration effects created by each motor are unequal
+            so that the effects of each can be combined to create more
+            complex haptic effects.
+          </p>
+          <p>
+            A {{GamepadHapticEffectType/"dual-rumble"}} effect is a
+            fixed-duration, constant-intensity vibration effect intended for an
+            actuator of this type. {{GamepadHapticEffectType/"dual-rumble"}}
+            effects are defined by {{GamepadEffectParameters/startDelay}},
+            {{GamepadEffectParameters/duration}},
+            {{GamepadEffectParameters/strongMagnitude}}, and
+            {{GamepadEffectParameters/weakMagnitude}}, none of which are
+            required because they default to 0.
+          </p>
+          <p>
+            {{GamepadEffectParameters/strongMagnitude}} and
+            {{GamepadEffectParameters/weakMagnitude}} set the
+            intensity levels for the low-frequency and high-frequency
+            vibrations, normalized to the range `[0,1]`, defaulting to 0.
+          </p>
+          <p>
+            Given {{GamepadEffectParameters}} |params:GamepadEffectParameters|,
+            a <dfn>valid dual-rumble effect</dfn> must have a valid
+            duration, a valid startDelay, and both the
+            {{GamepadEffectParameters/strongMagnitude}} and the
+            {{GamepadEffectParameters/weakMagnitude}} must be in the range
+            `[0,1]`.
+          </p>
+        </dd>
+      </dl>
+    </section>
+    <section>
+      <h3>
+        <dfn>GamepadEffectParameters</dfn> Dictionary
+      </h3>
+      <p>
+        A <code>GamepadEffectParameters</code> dictionary contains keys for
+        parameters used by haptic effects. The meaning of each key is defined
+        by the haptic effect, and some keys may be unused.
+      </p>
+      <p>
+        To mitigate unwanted long-running effects, the [=user agent=] MAY
+        limit the total effect duration for a [=valid effect=] to some
+        maximum duration. It is RECOMMENDED that the [=user agent=] use a
+        maximum of 5 seconds.
+      </p>
+      <pre class="idl">
+        dictionary GamepadEffectParameters {
+            unsigned long long duration = 0;
+            unsigned long long startDelay = 0;
+            double strongMagnitude = 0.0;
+            double weakMagnitude = 0.0;
+        };
+      </pre>
+      <dl>
+        <dt>
+          <dfn>duration</dfn> member
+        </dt>
+        <dd>
+          {{GamepadEffectParameters/duration}} sets the duration of the
+          vibration effect in milliseconds. An effect with an invalid
+          {{GamepadEffectParameters/duration}} is not a [=valid effect=].
```

Comment:
We don't need this, IMO:

```suggestion
          vibration effect in milliseconds.
```